doc/news/new_opmath.rst

```diff
@@ -32,15 +32,12 @@ Summary of the update
 
 .. rst-class:: admonition tip
 
-    The opt-in feature ``qml.operation.enable_new_opmath()`` is now the default. Ideally, your code should not break.
-    If it still does, it likely only requires some minor changes. For that, see the :ref:`Troubleshooting_opmath` section.
-    You can still opt-out and run legacy code via ``qml.operation.disable_new_opmath()``, though it is deprecated, and thus,
-    not recommended.
-
-.. warning::
-
-    In PennyLane v0.39, legacy operator arithmetic is deprecated and will be removed in v0.40.
-
+    After a period of deprecation, the legacy behaviour for operators was removed in PennyLane version 0.40.
+    Anyone using the latest PennyLane will automatically use the updated operator arithmetic. Ideally, your code
+    should not break when making this update. If it still does, it likely only requires some minor changes.
+    For that, see the :ref:`Troubleshooting_opmath` section. If you were using any of the functions explictly
+    provided to continue using the deprecated behaviour, like ``qml.operation.disable_new_opmath()``, that

```

Comment:
Will `disable_new_opmath` be available at all in v0.40? 🤔

Reply:
No, it won't - do you think that is unclear here?

Reply:
I'm reading this back and I don't understand where my confusion stemmed from... chalk it up to amnesia!

Feel free to resolve :)